db/re/item_db.txt

```diff
@@ -11534,14 +11597,139 @@
+23545,Silver_Statue,Silver Statue,11,,,1,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23545; },{},{}
+23546,Cursed_Blood,Cursed Blood,11,,,1,,,,,0xFFFFFFFF,63,2,,,,,,{ upgradeui 23546; },{},{}
+23547,Gold_Statue,Gold Statue,11,,,1,,,,,0xFFFFFFFF,63,2,,,,,,{ upgradeui 23547; },{},{}
+23629,Costume_Enchant_Stone_Box14,Costume Enchant Stone Box14,18,,,0,,,,,0xFFFFFFFF,63,2,,,,,,{ /*getgroupitem(IG_Costume_Enchant_Stone_Box14);*/ },{},{}
+23665,PhysicalMagical_Mix,PhysicalMagical Mix,11,,,1,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23665; },{},{}
+23666,ImmunedAthena_Mix,ImmunedAthena Mix,11,,,1,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23666; },{},{}
+23667,HardChamption_Mix,HardChampion Mix,11,,,1,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23667; },{},{}
+23668,KingbirdAncient_Mix,KingbirdAncient Mix,11,,,1,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23668; },{},{}
+23669,CriticalHit_Mix,CriticalHit Mix,11,,,1,,,,,0xFFFFFFFF,63,2,,,,,,{ synthesisui 23669; },{},{}
+23675,Geffen_Magic_Scroll,Geffen Magic Armor Order Form,11,,,1,,,,,0xFFFFFFFF,63,2,,,,,,{ upgradeui 23675; },{},{}
+23676,Gray_Charcoal_Melee,Grey Abrasive (physical),11,,,1,,,,,0xFFFFFFFF,63,2,,,,,,{ upgradeui 23676; },{},{}
+23677,Gray_Charcoal_Magic,Grey Abrasive (magical),11,,,1,,,,,0xFFFFFFFF,63,2,,,,,,{ upgradeui 23677; },{},{}
+23678,Gray_Charcoal_Range,Grey Abrasive (distance),11,,,1,,,,,0xFFFFFFFF,63,2,,,,,,{ upgradeui 23678; },{},{}
+23679,Geffen_Magic_Scroll2,Geffen Magic Accessory Order Form,11,,,1,,,,,0xFFFFFFFF,63,2,,,,,,{ upgradeui 23679; },{},{}
+23682,Costume_Enchant_Stone_Box15,Costume Enchant Stone Box15,18,,,0,,,,,0xFFFFFFFF,63,2,,,,,,{ /*getgroupitem(IG_Costume_Enchant_Stone_Box15);*/ },{},{}
+23706,Charleston_Parts_W,Charleston Upgrade Parts (Physical),11,,,1,,,,,0xFFFFFFFF,63,2,,,,,,{ upgradeui 23706; },{},{}
+23707,Charleston_Parts_R,Charleston Upgrade Parts (Ranged),11,,,1,,,,,0xFFFFFFFF,63,2,,,,,,{ upgradeui 23707; },{},{}
+23720,Shadow_Random_Mix,Shadow Random Mix,11,,,1,,,,,0xFFFFFFFF,63,2,,,,,,{ upgradeui 23720; },{},{}
+23770,Costume_Enchant_Stone_Box16,Costume Enchant Stone Box16,18,,,0,,,,,0xFFFFFFFF,63,2,,,,,,{ /*getgroupitem(IG_Costume_Enchant_Stone_Box16);*/ },{},{}
+23776,EP17_1_SPC05,Weapon Modifier (Physical),11,,,1,,,,,0xFFFFFFFF,63,2,,,,,,{ upgradeui 23776; },{},{}
+23777,EP17_1_SPC06,Advanced Weapon Modifier (Physical),11,,,1,,,,,0xFFFFFFFF,63,2,,,,,,{ upgradeui 23777; },{},{}
+23778,EP17_1_SPC07,Super Weapon Modifier (Physical),11,,,1,,,,,0xFFFFFFFF,63,2,,,,,,{ upgradeui 23778; },{},{}
+23779,EP17_1_SPC08,Weapon Modifier (Magic),11,,,1,,,,,0xFFFFFFFF,63,2,,,,,,{ upgradeui 23779; },{},{}
+23780,EP17_1_SPC09,Advanced Weapon Modifier (Magic),11,,,1,,,,,0xFFFFFFFF,63,2,,,,,,{ upgradeui 23780; },{},{}
+23781,EP17_1_SPC10,Super Weapon Modifier (Magic),11,,,1,,,,,0xFFFFFFFF,63,2,,,,,,{ upgradeui 23781; },{},{}
+23815,Magma_Essence,Lava Essence,11,,,1,,,,,0xFFFFFFFF,63,2,,,,,,{ upgradeui 23815; },{},{}
```

Comment:
23815 target item not appearing. Other upgrade are working fine. Already sync with client files.

Reply:
only display equips with correct conditions, have 2 random options
![image](https://user-images.githubusercontent.com/4016864/65045060-549f0c80-d988-11e9-8706-e9b1af0987f5.png)

Reply:
@cydh Oh silly me. Apologies for this one. Because I didn't have yet the translation in the lua I thought I was doing it right.

Reply:
@ecdarreola

update your **lapine lub**

Reply:
> @ecdarreola
> 
> update your **lapine lub**

I have an updated one or already sync with @cydh PR. However the translation of the items are not (iteminfo.lua).